subtests/docker_cli/run/run_tmpfs.py

```diff
@@ -0,0 +1,38 @@
+from run import run_base
+
+
+class run_tmpfs(run_base):
+    # Verify behavior when multiple --name options passed
+
+    def init_subargs(self):
+        cont = self.sub_stuff["cont"]
+        name_base = cont.get_unique_name()
+        names = []
+        for number in xrange(self.config['names_count']):
+            name = ('%s_%d' % (name_base, number))
+            names.append(name)
+            self.sub_stuff['containers'].append(name)  # just in case
+        self.sub_stuff['subargs'] += ["--name %s" % name for name in names]
+        if self.config['last_name_sticks']:
+            self.sub_stuff['expected_name'] = names[-1]
+        else:
+            self.sub_stuff['expected_name'] = names[0]
+        super(run_tmpfs, self).init_subargs()
+
+    def run_once(self):
+        super(run_tmpfs, self).run_once()
+        cid = self.sub_stuff['cid'] = self.sub_stuff['dkrcmd'].stdout.strip()
+        self.sub_stuff['containers'].append(cid)
+        try:
+            self.sub_stuff["cont"].wait_by_long_id(cid)
+        except ValueError:
+            pass  # container already finished and exited
+
+    def postprocess(self):
+        super(run_tmpfs, self).postprocess()
```

Comment:
base class just does basic checks here for exit codes, panic messages, etc.  So you'll want to nuke most of the lines below here, and add your customized checks to verify the ``--tmpfs`` option functioned as expected.  This is where the real _meat_ of your test will go.

if you find the docker command executed (prior to getting here) is a PITA to verify (for --tmpfs), then step back and work on ``initialize()`` and/or ``run_once()`` further.   Bend the behaviour as needed, to make verification easier.  Lather...rinse...and repeat :D